graphql/database/invitedUsers/createInvitedUsers.js

```diff
@@ -0,0 +1,86 @@
+import moment from 'moment'
+import uniq from 'lodash/uniq'
+import xssFilters from 'xss-filters'
+import InvitedUsersModel from './InvitedUsersModel'
+import UserModel from '../users/UserModel'
+import { verifyAndSendInvite } from './utils'
+import {
+  ADMIN_MANAGEMENT,
+  getPermissionsOverride,
+} from '../../utils/permissions-overrides'
+
+/**
+ * conditionally creates a new invited user and sends a sendgrid email if user has not already
+ * been invited OR if inviting user has exceeded the max amount of invites in 24 hours
+ * @param {object} userContext - The user authorizer object.
+ * @param {string} inviterId - The user id.
+ * @param {array} invitedEmails - email addresses to send
+ * @param {string} inviterName - name to include in ivite email
+ * @param {string} inviterMessage - optional personal message to include in email
+ * @return {Promise<User>}  A promise that resolves into a invitedUser instance or an error.
+ */
+const createInvitedUsers = async (
+  userContext,
+  inviterId,
+  invitedEmails,
+  inviterName,
+  inviterMessage
+) => {
+  try {
+    const override = getPermissionsOverride(ADMIN_MANAGEMENT)
+    const endTimeRoundedISO = moment()
+      .startOf('hour')
```

Comment:
No need to round the hour, right?

Reply:
no need, i just like comparing times with 0s.